packages/base-zone/src/prepare-revocable.js

```diff
@@ -4,14 +4,22 @@ import { fromUniqueEntries } from '@endo/common/from-unique-entries.js';
 const { Fail, quote: q } = assert;
 
 /**
- * @typedef {object} Revoker
- * @property {() => boolean} revoke
+ * @template {any} [U=any]
+ * @typedef {object} RevocableMakerKit
+ * @property {(revocable: U) => boolean} revoke
+ * @property {(underlying: U) => U} makeRevocable
+ *   Forwards to the underlying exo object, until revoked
+ */
+
+/**
+ * @typedef {object} RevokerFacet
+ * @property {() => boolean} revokeOne
```

Comment:
At first glance, I don't understand why this method is called `revokeOne`.  Can you add some docs?

Reply:
The distinction I was trying to make wasn't worth explaining. I changed it to just `revoke`. 

That distinction is that it doesn't take an argument because it revokes only one thing, which is the cohort that it is a member of. As opposed to the exposed `revoke(revocable)` on the maker kit, that can revoke an revocable made by this maker.

Done.